mne/beamformer/_lcmv.py

```diff
@@ -655,11 +654,7 @@ def lcmv(evoked, forward, noise_cov=None, data_cov=None, reg=0.05, label=None,
 
     Notes
     -----
-    The original reference is [1]_
-    The reference for finding the max-power orientation is:
-    Sekihara et al. Asymptotic SNR of scalar and vector minimum-variance
-    beamformers for neuromagnetic source reconstruction.
-    Biomedical Engineering (2004) vol. 51 (10) pp. 1726--34
```

Comment:
That's refers to the source orientation formula we deleted (right now, only max-power with weight normalization is supported).